### PR TITLE
Updates to the release notes for 0.33

### PR DESCRIPTION
**Pull request by trbromley**

### CODE REVIEW

File: doc/releases/changelog-dev.md [hunk 1]
```diff
@@ -56,36 +31,89 @@
...
+    and hence does not need to be swapped:
+
+    ```pycon
+    >>> f.device
+    <default.qubit device (wires=1) at 0x7f20d043b040>
+    >>> f.device == dev
+    True
+    ```
+
+    * TODO
```
Comment (trbromley) [inline doc/releases/changelog-dev.md:87]: @albi3ro let's work together on a nice bullet point for the VJP/JVP work.

File: doc/releases/changelog-dev.md [hunk 2]
```diff
@@ -4,192 +4,331 @@
...
 
-* Support drawing QJIT QNode from Catalyst.
-  [(#4609)](https://github.com/PennyLaneAI/pennylane/pull/4609)
+<h4>Decompose circuits into the Clifford+T gateset 🧩</h4>
 
-  ```python
-  import catalyst
+* A new transform called `clifford_t_decomposition` is available, which decomposes
+  circuits into the Clifford+T gate set. 
+  [(#ABCD)]()
```
Comment (isaacdevlugt) [inline doc/releases/changelog-dev.md:11]: Just putting a note here that the PR numbers need to be added once they're merged

File: doc/releases/changelog-dev.md [hunk 3]
```diff
@@ -4,192 +4,331 @@
...
 
-* Transforms can be applied on devices following the new device API.
- [(#4667)](https://github.com/PennyLaneAI/pennylane/pull/4667)
+<h4>Exponentiate Hamiltonians with flexible Trotter products 🐖</h4>
 
-* All gradient transforms are updated to the new transform program system.
- [(#4595)](https://github.com/PennyLaneAI/pennylane/pull/4595)
+* Higher-order Trotter-Suzuki methods are now easily accessible through a new operation
+  called `TrotterProduct`.
+  [(#ABCD)]()
```
Comment (isaacdevlugt) [inline doc/releases/changelog-dev.md:77]: Just putting a note here that the PR numbers need to be added once they're merged

File: doc/releases/changelog-dev.md [hunk 4]
```diff
@@ -4,7 +4,151 @@
...
+        qml.RZ(0.1, wires="device")
+        qml.Hadamard("api")
+        return qml.state()
+    ```
+
+    ```pycon
+    >>> print(qml.draw(circuit)())
+       new: ──Z────────┤  State
+    device: ──RZ(0.10)─┤  State
+       api: ──H────────┤  State
```
Comment (isaacdevlugt) [inline doc/releases/changelog-dev.md:113]: 🤠 

a little on the nose but lemme know what you think haha

Comment (trbromley) [reply doc/releases/changelog-dev.md:113]: I think it's fun but worry it detracts from the point we're trying to make. E.g., does it now look like something about custom wire labels?

File: doc/introduction/measurements.rst [hunk 5]
```diff
@@ -385,13 +385,17 @@ Mid-circuit measurement statistics
 
 Statistics can be collected on mid-circuit measurements along with terminal measurement statistics.
 Currently, ``qml.probs``, ``qml.sample``, ``qml.expval``, ``qml.var``, and ``qml.counts`` are supported,
+<<<<<<< HEAD
+and can be requested along with other measurements.
+=======
```
Comment (isaacdevlugt) [inline doc/introduction/measurements.rst:390]:
```suggestion
```

File: doc/introduction/measurements.rst [hunk 6]
```diff
@@ -385,13 +385,17 @@ Mid-circuit measurement statistics
...
 and can be requested along with other measurements. The devices that currently support collecting such
 statistics are :class:`~.pennylane.devices.DefaultQubit`, :class:`~.pennylane.devices.DefaultMixed`, and
 :class:`~.pennylane.devices.DefaultQubitLegacy`.
+>>>>>>> master
```
Comment (isaacdevlugt) [inline doc/introduction/measurements.rst:394]:
```suggestion
```

File: doc/introduction/measurements.rst [hunk 7]
```diff
@@ -406,7 +410,18 @@ Executing this QNode:
 (tensor([0.9267767, 0.0732233], requires_grad=True),
  tensor([0.5, 0.5], requires_grad=True))
 
+<<<<<<< HEAD
```
Comment (isaacdevlugt) [inline doc/introduction/measurements.rst:413]:
```suggestion
```

File: doc/introduction/measurements.rst [hunk 8]
```diff
@@ -406,7 +410,18 @@ Executing this QNode:
...
+application of mid-circuit measurements and conditional operations in a
+differentiable and device-independent way. Performing true mid-circuit
+measurements and conditional operations is dependent on the
+quantum hardware and PennyLane device capabilities.
+
+For more examples on applying quantum functions conditionally, refer to the
+:func:`~.pennylane.cond` transform.
+=======
 .. warning::
+>>>>>>> master
```
Comment (isaacdevlugt) [inline doc/introduction/measurements.rst:424]:
```suggestion
```

File: doc/introduction/measurements.rst [hunk 9]
```diff
@@ -406,7 +410,18 @@ Executing this QNode:
...
+<<<<<<< HEAD
+The deferred measurement principle provides a natural method to simulate the
+application of mid-circuit measurements and conditional operations in a
+differentiable and device-independent way. Performing true mid-circuit
+measurements and conditional operations is dependent on the
+quantum hardware and PennyLane device capabilities.
+
+For more examples on applying quantum functions conditionally, refer to the
+:func:`~.pennylane.cond` transform.
+=======
```
Comment (isaacdevlugt) [inline doc/introduction/measurements.rst:422]:
```suggestion

```

File: doc/releases/changelog-0.33.0.md [hunk 10]
```diff
@@ -191,47 +147,142 @@
   plt.bar(range(len(output)), output)
   plt.show()
   ```
+  
+  <img src="_images/cosine_window.png" width=50%/>
```
Comment (isaacdevlugt) [inline doc/releases/changelog-0.33.0.md:151]: Not sure how to add this picture but I think it's beneficial to have 
cosine_window

File: doc/releases/changelog-0.33.0.md [hunk 11]
```diff
@@ -191,47 +147,142 @@
   plt.bar(range(len(output)), output)
```
Comment (isaacdevlugt) [inline doc/releases/changelog-0.33.0.md:147]:
```suggestion
  plt.style.use("pennylane.drawer.plot")
  plt.bar(range(len(output)), output)
```

File: doc/releases/changelog-0.33.0.md [hunk 12]
```diff
@@ -188,50 +144,146 @@
...
 
-  dev = qml.device("default.qubit", wires=5)
+  This changeover has a number of benefits for `default.qubit`, including:
 
-  op = qml.CosineWindow(wires=range(5))
-  
-  @qml.qnode(dev)
-  def circuit():
-      op.decomposition()
-      return qml.state()
```
Comment (isaacdevlugt) [inline doc/releases/changelog-0.33.0.md:207]: Took this part out of the CosineWindow entry. Should we show its decomposition?

**Comment by trbromley** [comment]
Replaced by https://github.com/PennyLaneAI/pennylane/pull/4720 and https://github.com/PennyLaneAI/pennylane/pull/4721.

File: doc/releases/changelog-0.33.0.md (hunk 12 above)
Comment (trbromley) [reply doc/releases/changelog-0.33.0.md]: I can't get the decomp to show here easily. Let's leave it out for now but another datapoint for better control over decomposition!
